linkerd/core/src/test/scala/io/buoyant/linkerd/RouterTest.scala

```diff
@@ -96,4 +102,23 @@ servers:
     val DstBindingFactory.Namer(interpreter) = router.params[DstBindingFactory.Namer]
     assert(interpreter.isInstanceOf[TestInterpreter])
   }
+
+  test("with retries") {
+    val yaml =
+      """|protocol: plain
+         |client:
+         |  retries:
+         |    backoff:
+         |      kind: jittered
+         |      maxMs: 10000
```

Comment:
Related to my comment below, I'm surprised that this config is valid without minMs specified.  Given that the jittered backoff case class is defined as `JitteredBackoffConfig(minMs: Int, maxMs: Int)`, I thought that implies that both params are required -- what am I missing?